### PR TITLE
Fix write_crystfel_geom offset unit

### PR DESCRIPTION
**Pull request by JunCEEE**
Fix issue #101

### CODE REVIEW

**Comment by JunCEEE** [comment]
Please see the current workaround:
https://github.com/European-XFEL/EXtra-geom/pull/102/files

File: extra_geom/crystfel_fmt.py [hunk 1]
```diff
@@ -74,7 +75,7 @@ def frag_to_crystfel(fragment, p, a, ss_slice, fs_slice, dims, pixel_size):
         fs_vec=_crystfel_format_vec(fragment.fs_vec/ pixel_size),
         corner_x=c[0],
         corner_y=c[1],
-        coffset=c[2],
+        coffset=coffset,
```
Comment (takluyver) [inline extra_geom/crystfel_fmt.py:78]:
```suggestion
        coffset=fragment.corner_pos[2],
```

No need for an extra variable. :slightly_smiling_face:

File: extra_geom/crystfel_fmt.py [hunk 2]
```diff
@@ -54,7 +54,8 @@ def _crystfel_format_vec(vec):
 
 def frag_to_crystfel(fragment, p, a, ss_slice, fs_slice, dims, pixel_size):
     tile_name = 'p{}a{}'.format(p, a)
-    c = fragment.corner_pos / pixel_size
+    c = fragment.corner_pos[:2] / pixel_size
+    coffset = fragment.corner_pos[2]
```
Comment (takluyver) [inline extra_geom/crystfel_fmt.py:58]:
```suggestion
```

**Review by takluyver — COMMENTED** [review]
Thanks Jun, this looks fine. I've made a little simplification, which I'll apply now and merge it.

**Pull request merged**